modules/lambda/lambda.tf

```diff
@@ -49,7 +49,7 @@ data "aws_iam_policy_document" "assume_role_policy" {
   }
 }
 
-resource "aws_iam_role" "lambda_execution_role" {
+resource "aws_iam_role" "lambda_role" {
```

Comment:
changing the resource name will effect on users that already used the module. it will delete and recreate the resource.